LDAP-Auth/Api/LdapController.cs

```diff
@@ -44,9 +60,97 @@ public IActionResult TestServerBind([FromBody] ServerConnectionInfo body)
             configuration.LdapBaseDn = body.LdapBaseDn;
             LdapPlugin.Instance.UpdateConfiguration(configuration);
 
-            var result = LdapAuthenticationProviderPlugin.TestServerBind();
+            var result = _ldapAuthenticationProvider.TestServerBind();
+
+            return Ok(new { Result = result });
```

Comment:
I'd prefer to not use an anonymous object for api return
(same for all endpoints)

Reply:
I've been using the other plugins for examples but that's where I got the anonymous object pattern. If the new version isn't what you wanted pointing me to somewhere that does it right would be helpful.

Reply:
Your implementation is perfect, the other plugins haven't had a full pass done on them yet so they still use some poor pattern